routes/api/shoppers.js

```diff
@@ -2,66 +2,87 @@ const express = require('express');
 const HttpStatus = require('http-status-codes');
 const router = express.Router();
 
+const interceptors = require('../interceptors');
```

Comment:
Note that I'm now using request interceptors to check whether the logged in user has the privileges to execute certain actions.

For example, to "list all shoppers" requires the user to be an Admin. You create admin users using the bin/create-admin script run inside the server container.